src/Utils/OrientationUtils.js

```diff
@@ -160,10 +186,13 @@ export default {
      * @param {Projection} proj the lcc projection (may be parsed using proj4)
      * @param {number} proj.lat0 - the latitude of origin
      * @param {number} proj.long0 - the longitude of the central meridian
-     *
-     * @return {OrientationFunction} the orientation function
+     * @param {Coordinates} [coordinates]  coordinates the origin of the local East North Up
+     * (ENU) frame
+     * @param {THREE.Quaternion} [target=new THREE.Quaternion()] output Quaternion
+     * @return {FunctionOrQuaternion} The target quaternion if coordinates is defined, otherwise, a function to compute it from coordinates.
     */
-    quaternionToLCC(proj) {
+    quaternionFromLCC(proj, coordinates, target = new THREE.Quaternion()) {
```

Comment:
The parameter `proj` may not be useless if we use the `crs coordinates` to find it.